lib/ansible/modules/cloud/google/gcp_compute_network_facts.py

```diff
@@ -47,6 +47,10 @@
            U(https://cloud.google.com/sdk/gcloud/reference/topic/filters).
            Each additional filter in the list will act be added as an AND condition
            (filter1 and filter2)
+    scopes:
```

Comment:
This won't work for the documentation. scopes is already included as a doc fragment. Line 54 has 'extends_documentation_fragment: gcp.  You can find that file at https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/gcp.py  and info on doc fragments in general at https://docs.ansible.com/ansible/2.5/dev_guide/developing_modules_documenting.html#documentation-fragments.

Reply:
I will add this comment back on the issue itself, but I'm not sure if this is the correct fix overall.  scopes is a common parameter across most if not all of  the gcp_* modules.  Further in this file  at line 157 is the following code: 
   if 'scopes' not in module.params:
module.params['scopes'] = ['https://www.googleapis.com/auth/compute']

That suggests to me that the parameter is not required and the code will fill in this value if it's not present. (Take that w/ a grain o salt as I don't write the code, so I could be missing something obvious here).

Reply:
@ryansb @bcoca  - need your opinions here.  This doc PR isn't correct 'as is' but from the main issue, it seems like the code isn't working as expected for the 'scopes' parameter. Since it's part of a doc fragment for most (all?) gcp_* modules, I'm not sure what the correction should be.

Reply:
> This won't work for the documentation. scopes is already included as a doc fragment. 

This will produce the same documentation present in i.e. `[gcp_compute_firewall](https://docs.ansible.com/ansible/2.6/modules/gcp_compute_firewall_module.html)` with the `scopes` marked as required (which, BTW, _also_ highlights docs inconsistencies).

Happy to make whichever changed you guys think is required.